M/MPICH/build_tarballs.jl

```diff
@@ -11,7 +11,7 @@ script = raw"""
 # Enter the funzone
 cd ${WORKSPACE}/srcdir/mpich-*
 
-./configure --prefix=$prefix --host=$target --enable-shared=yes --enable-static=no --disable-dependency-tracking --disable-fortran --docdir=/tmp
+./configure --prefix=$prefix --host=$target --enable-shared=yes --enable-static=no --disable-dependency-tracking --disable-fortran --docdir=/tmp --with-file-system=testfs+ufs+nfs
```

Comment:
In `src/mpi/romio/configure` there is
```sh
#
# if FILE_SYSTEM is not set above, use ufs and nfs as default
#
if test -z "$FILE_SYSTEM" ; then
    FILE_SYSTEM="ufs nfs"
fi

# no matter what, always build testfs
FILE_SYSTEM="testfs $FILE_SYSTEM"
```
so my understanding is that this should be
```suggestion
./configure --prefix=$prefix --host=$target --enable-shared=yes --enable-static=no --disable-dependency-tracking --disable-fortran --docdir=/tmp --with-file-system="testfs ufs nfs"
```
but this list list is already the default.  What do you think?

Reply:
I don't know, I did some tests locally, I couldn't find a way to convince `configure` to accept the file systems passed from the command line :confused: